src/frontend.js

```diff
@@ -1,7 +1,7 @@
 import Webamp from 'webamp';
-import domReady from '@wordpress/dom-ready';
```

Comment:
`@wordpress/dom-ready` is no more a dependency for this and other scripts, could be removed from `package.json`

Reply:
@iansvo if you concur with ^, mind a quick commit to pull that out?

Reply:
@cadic could you handle that update and then look to get this merged in?

Reply:
@cadic @jeffpaul Update is made confirmed build succeeds, merged in latest from dev and regenerated package-lock.json.